.github/workflows/hecke.yml

```
@@ -103,6 +104,9 @@ jobs:
                                            varname=\"oscar_run_doctests\",
                                            filename=\"${GITHUB_ENV}\");"
 
+      - name: "workaround libstdc++ issue for julia 1.6"
+        if: matrix.julia-version == '1.6' && runner.os == 'Linux'
```

```suggestion
        if: matrix.julia-version == '1.6' # && runner.os == 'Linux'
```
this job only runs on Linux